Api/ATM/ATM1/Response.php

```diff
@@ -49,8 +49,9 @@ public function __construct($response)
         parent::__construct($response);
 
         $responseObject = new \SimpleXMLElement($response);
-        $this->returnCode = (int)$responseObject->retval;
-        $this->returnDescription = (string)$responseObject->retdesc;
+        
+        $this->returnCode        = (int) $responseObject->retval;
+        $this->returnDescription = (string) $responseObject->retdesc . '. '. (string) $responseObject->description;
```

Comment:
@baibaratsky need to reformat this after merging. I see not enough space and unacceptable spaces before `=`